setup.py

```diff
@@ -12,13 +12,20 @@
 import sys
 import setuptools
 
-ctre_lib_version = "5.14.0"
```

Comment:
Should keep 5.14 unless you have a good reason?

Reply:
Odd, it was 5.12.1.  The raspbian binaries aren't available before 5.13.0 and Dustin said you were due to move up so I changed it to 5.13.  I guess a change to 5.14 came in when I wasn't looking, though I'm surprised git didn't yell at me about that.  Mea culpa, glad to change it to 5.14.

Reply:
They technically are available before 5.13.0, it's just that the platform tag for the artifacts got renamed to reflect WPILib's naming.

Reply:
Okay, I committed a change that includes changing it back to 5.14.0.